src/components/auth/AuthHeader.tsx

```diff
@@ -6,7 +6,9 @@ import React from "react";
 import { Button } from "@/components/ui/buttons/button";
 
 const AuthHeader = () => {
-  const { isLoggedIn, isLoading } = useAppContext();
+  const context = useAppContext();
+  const isLoggedIn = context?.isLoggedIn;
+  const isLoading = context?.isLoading;
```

Comment:
As it was destructured already, any reason to spread that again ? 

`const { isLoggedIn, isLoading } = useAppContext();`

Reply:
Ah this was unnecessary, I have reverted my change 👍